packages/pagebuilder/lib/ContentTypes/Banner/banner.js

```diff
@@ -92,7 +96,8 @@ const Banner = props => {
         borderColor,
         borderWidth,
         borderRadius,
-        textAlign
+        textAlign,
+        ...(appearance !== 'poster' && mediaQueryStyles)
```

Comment:
Why wouldn't we want other banner types to have the mobile styles?

Reply:
You're right this shouldn't be here, I forgot to remove it.